src/sql/media/icons/run-with-parameters.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I see you have a copy of this SVG in another location.
Please use exactly one SVG.